langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -215,6 +303,18 @@ public isolated function getContent(ProcessingInstruction|Comment x) returns str
 
 # Creates a new xml element item.
 #
+# ```ballerina
+# xml:createElement(
+#     "book", 
+#     {title: "The Adventures of Sherlock Holmes", year: "1892"}, 
+#     xml `<author>Arthur Conan Doyle</author>`
+# ) ⇒ <book title="The Adventures of Sherlock Holmes" year="1892"><author>Arthur Conan Doyle</author></book>
+# 
+# xml:createElement("student") ⇒ <student/>
+# 
+# xml:createElement("employee", children = xml `<name>John</name>`) ⇒ <employee><name>John</name></employee>
```

Comment:
Shall we add one with only attributes and name

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/e8373b00542df5fae76b3a78e7b77f28c6a896c0